### ISSUE TITLE
Add a delete to sub doesn't exist bot

### ISSUE THREAD

**Issue by JustCallMeRay** [Toldry/RedditAutoCrosspostBot#2]
It kinda ruins subreddits like r/subsIfellfor, have the bot delete it's comment if negitive karma or have the parent commenter be able to delete it (preferably via a message but a link in comment works too)

**Comment by Toldry** [comment]
Good idea, I'll put it in my backlog.

Do you perhaps know of a bot that already has this feature so I could easily copy-paste the code?

**Comment by JustCallMeRay** [comment]
Sneak peak bot has a blacklist (which I'd also appreciate) and should be easy to implement. 

Backtick bot has a comment reply based blacklist.

**Comment by Toldry** [comment]
Do you have Backtick bot's source code? I couldn't find it by googling

I did find Sneak peak bot's code here:
https://github.com/fterh/sneakpeek

**Comment by Toldry** [comment]
Is this the blacklist you mentioned in Sneak peak bot's code?:

https://github.com/fterh/sneakpeek/blob/5e25bae6ec14c714c964b7558795d4cdc91ba714/handlers/straits_times_handler/__init__.py#L187

This code seems unrelated to implementing a solution of the issue you opened, unless I am misunderstanding something. If so, please clarify.

**Comment by JustCallMeRay** [comment]
I don't know python (at all) that bot looks completely unrelated to u/sneakpeakbot. Just DM sneakpeakbot and ask how he did it, he's often online by the looks of it

**Comment by Toldry** [comment]
sent them a message, waiting for a response

**Comment by JustCallMeRay** [comment]
Cool, 
Again I dont know python but from memory of making a java bot it was just get the comment author and then check them against a blacklist, for now you can update the blacklist manually.

Also I found [this] (https://github.com/Cyb3r-Jak3/useful_bot/tree/master/useful_bot)

**Comment by KRA2008** [comment]
I agree with OP but would go further to say this bot simply should not exist, see my expanded explanation in #13

**Comment by Toldry** [comment]
u/sneakpeakbot responded, this is the repo

https://github.com/ghnr/sneakpeekbot

i'll use this later

**Comment by Kulkinz** [comment]
Would like to second the blacklist idea. An ability to opt out of the bot responding to user comments would be greatly preferred, if want you keep the bot running

**Comment by abraxas86** [comment]
Or just have it DM the user who made the post with a link to the comment and the heads-up that the sub that they entered doesn't exist

**Comment by Toldry** [comment]
That's a good idea @abraxas86 but part of my intention for the bot's function is to notify not just the user who posted the nonexistent subreddit but also to other users who happ upon the comment, so they won't waste time clicking on a link to a subreddit that doesn't exist, while also informing them of other subreddits with similar names
